src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/1097d30f85.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>To do-list</title>
+  </head>
+  <body>
+    <form class="todo-form">
+      <div class="title">
+        <p>Today's To Do</p>
+        <button class="restart" type="button">
+          <i class="fas fa-sync-alt"></i>
+        </button>
+      </div>
+      <div class="add-list">
+        <input type="text" name="name" id="task" 
+        autofocus placeholder="Add to your list..." />
+        <button type="submit" class="btn-add">
+          <i class="fas fa-level-down-alt"></i>
+        </button>
+      </div>
+      <ul class="todo-items"></ul>
+      <button class="clear-all" type="button">Clear all completed</button>
```

Comment:
- [ ] [OPTIONAL] The div with class title, ul tag, and the button with class clear-all should be outside the form tag